Package.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  Package.swift
+//  Yoshi
+//
+//  Created by Michael Campbell on 4/15/16.
+//  Copyright © 2016 Prolific Interactive. All rights reserved.
+//
+
+import PackageDescription
+
+let package = Package(
+    name: "Yoshi"
```

Comment:
If I'm not mistaken, this should look more like this:
```swift
    name: "Yoshi",
    targets: [
        Target(
            name: "Yoshi"
        ),
        Target(
            name: "YoshiTests",
            dependencies: [.Target(name: "Yoshi")]
        )
    ]
```
This should create separate targets for the actual module and test targets.
The output currently: 
```shell
.../Yoshi.swift:9:8: error: no such module 'UIKit'
import UIKit
       ^
.../Yoshi.swift:9:8: error: no such module 'UIKit'
import UIKit
       ^
.../Yoshi.swift:9:8: error: no such module 'UIKit'
import UIKit
       ^
.../Yoshi.swift:9:8: error: no such module 'UIKit'
import UIKit
       ^
.../Yoshi.swift:9:8: error: no such module 'UIKit'
import UIKit
       ^
.../Yoshi.swift:9:8: error: no such module 'UIKit'
import UIKit
       ^
.../Yoshi.swift:9:8: error: no such module 'UIKit'
import UIKit
       ^
.../Yoshi.swift:9:8: error: no such module 'UIKit'
import UIKit
       ^
.../YoshiTests.swift:10:18: error: no such module 'Yoshi'
@testable import Yoshi
                 ^
<unknown>:0: error: build had 2 command failures
```

Where "Yoshi" seems to be referenced from within itself. When changing it as mentioned above, you get this:

```shell
.../Yoshi.swift:9:8: error: no such module 'UIKit'
import UIKit
       ^
.../Yoshi.swift:9:8: error: no such module 'UIKit'
import UIKit
       ^
.../Yoshi.swift:9:8: error: no such module 'UIKit'
import UIKit
       ^
.../Yoshi.swift:9:8: error: no such module 'UIKit'
import UIKit
       ^
.../Yoshi.swift:9:8: error: no such module 'UIKit'
import UIKit
       ^
.../Yoshi.swift:9:8: error: no such module 'UIKit'
import UIKit
       ^
.../Yoshi.swift:9:8: error: no such module 'UIKit'
import UIKit
       ^
.../Yoshi.swift:9:8: error: no such module 'UIKit'
import UIKit
       ^
<unknown>:0: error: build had 1 command failures
```

Where the "no such module errors" are caused by a [lack of support for iOS so far](https://github.com/apple/swift-package-manager/blob/master/Documentation/Package.swift.md#depending-on-apple-modules-eg-foundation).